.stickler.yml

```diff
@@ -1,3 +1,3 @@
 linters:
-stylelint:
-config: ‘stylelint.config.js’ 
\ No newline at end of file
+  stylelint:
+    config: ‘stylelint.config.js’ 
```

Comment:
Let's add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).